doc/modules/mod_http_upload.md

```diff
@@ -0,0 +1,37 @@
+### Module Description
```

Comment:
Please add the file to `mkdocs.yml` otherwise it will not be added to the doc on readthedocs.io

Also, you can check the documentation locally by running 
```
mkdocs server
```

from root directory. More info about mkdocs is here: http://www.mkdocs.org